src/ur_arm.cpp

```diff
@@ -147,6 +147,7 @@ struct URArm::state_ {
     std::atomic<double> speed{0};
     std::atomic<double> acceleration{0};
     std::atomic<bool> estop{false};
+    std::atomic<bool> local_disconnect{false};
```

Comment:
We now have 4 atomic bools tracking the runtime state of the arm, which is 16 possible combinations, only a small subset of which are actually legit. I don't want to address it in this review, since this isn't the review in which we should be making major structural changes, but a better approach is needed.